src/platform/qt/AudioDevice.cpp

```diff
@@ -63,12 +63,17 @@ qint64 AudioDevice::writeData(const char*, qint64) {
 	return 0;
 }
 
+bool AudioDevice::isSequential() const {
+	return true;
+}
+
 bool AudioDevice::atEnd() const {
-	if (!m_context->core) {
-		return true;
-	}
+	return bytesAvailable() != 0;
```

Comment:
This check appears to be backwards.

```suggestion
	return !bytesAvailable();
```